packages/components/src/theme/ComposerTheme/Components/Button.ts

```diff
@@ -120,6 +127,7 @@ export const Button: ComponentStyleConfig = {
         height: $spinnerSize.reference,
         padding: 0,
       },
+      fontSize: '0.875rem',
```

Comment:
I don't think that's the right solution, the button should be using a text variant that should match the design. We shouldn't define specific fontSize

Reply:
yep, ideally it should take that font size from `textStyles` and by variant, the font style is correct according to the design but it does not match the font size, so I thought that maybe it is a particular case and instead of modifying the font style and affecting other components I decided to change it only here where I saw the error. But it could be an inconsistency in the design or that the font size of that specific text style has been changed.

Reply:
So probably we should update the `text-body-meta-medium` to have the design fontSize, it should be using 0.875 instead of 0.75, right?

Reply:
Okay, cool if you guys agree, I am gonna make this change.